frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -124,10 +272,11 @@ class DashboardInner extends Component {
         addCardToDashboard({
           dashId: dashboardId,
           cardId: addCardOnLoad,
-          tabId: this.props.dashboard.tabs[0]?.id ?? null,
+          tabId: this.props.dashboard.tabs?.[0]?.id ?? null,
         });
       }
-    } catch (error) {
+    } catch (err) {
+      const error = err as any;
       if (error.status === 404) {
```

Comment:
```suggestion
      if (error instance of Response && error.status === 404) {
```

or please add a custom type guard